recipe/build.sh

```diff
@@ -0,0 +1,10 @@
+# do not build winpty.exe as this requires a Cygwin/MSYS targetting toolchain
+./configure
+make build/winpty-agent.exe
+make build/winpty.dll
```

Comment:
Why are you not compilling the debugserver.exe?

Reply:
It didn't seem as if the debugserver executable was needed for pywinpty.  I can add it in if it is.

Reply:
Well, is better to have just in case